lib/src/object_group.dart

```diff
@@ -12,6 +12,14 @@ class ObjectGroup {
   Map<String, dynamic> properties = {};
   List<TmxObject> tmxObjects = [];
 
+  ObjectGroup();
```

Comment:
Removing it leads to a compile error.
The named constructor fromXML removes the default constructor. So i need it here.... or am i wrong?

Reply:
What kind of compile error, because I just did a quick test in dartpad and a named constructor is allowed without a default constructor.

Possibly that you are trying to construct these classes using the default constructor, but because they dont exist it gives a compile error?

Reply:
Yes. 
i do:
    final ObjectGroup objectGroup = ObjectGroup();
The error is, that ObjectGroup does not provide a default constructor

Reply:
You should not do it like that then. If you need a specific usecase for it, you should turn that usecase into a named constructor.

Reply:
I think the default constructor is ok if it's used, we have classes on flame with named + default constructors where it makes sense.

Reply:
old code, so i will resolve this